src/routes/Gdp.jsx

```diff
@@ -0,0 +1,27 @@
+import GdpPie from "../components/GdpPie";
+import FetchAPI from "../components/FetchAPI";
+import Loading from "../components/Loading";
+import Error from "../components/Error";
+
+export default function GDP() {
+  const { countries, loading, error } = FetchAPI();
+
+  if (loading) {
+    return <Loading />;
+  }
```

Comment:
I like the inclusion of a loading page here. I haven't seen it, but it would help with keeping any users from leaving the site due to impatience

Reply:
It should be some cyan text, centered, rendered before the content, and states ```Loading...```